docs/docsite/rst/dev_guide/testing_integration.rst

```diff
@@ -66,6 +66,8 @@ outside of those test subdirectories.  They will also not reconfigure or bounce
 
    To protect your system from any potential changes caused by integration tests, and to ensure a sensible set of dependencies are available we recommend that you always run integration tests with the ``--docker`` option. See the `list of supported docker images <https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/_data/completion/docker.txt>`_ for options.
 
+   Don't use the ``default`` docker image (used for sanity and unit tests only), always pass an image name explicitly.
```

Comment:
Minor niggle, from a 'teaching' point of view it's generally worth giving a 'positive' example rather than just a negative one.  The document you link to doesn't explain why you'd make a given choice, just that you have one.

Reply:
@tremble , good suggestion. Changed. PTAL